src/odemis/acq/fastem.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on 19 Apr 2021
+
+Copyright © 2021 Philip Winkler, Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms
+of the GNU General Public License version 2 as published by the Free Software
+Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+"""
+from odemis import model
+
+
+class FastEMROA(object):
+    """ Representation of a FastEM acquisition region. """
+
+    def __init__(self, name, coordinates, calibration):
+        self.name = model.StringVA(name)
+        self.coordinates = model.TupleContinuous(coordinates,
+                                                 range=((-1, -1, -1, -1), (1, 1, 1, 1)),
+                                                 cls=(int, float))
+        self.calibration = model.VigilantAttribute(calibration)  # ROC
+
+
+class FastEMROC(object):
+    """ Representation of a FastEM parameters region. """
+
+    def __init__(self, number, coordinates):
+        self.number = model.IntVA(number, readonly=True)  # cannot be changed after initialization
+        self.coordinates = model.TupleContinuous(coordinates,
+                                                 range=((-1, -1, -1, -1), (1, 1, 1, 1)),
+                                                 cls=(int, float))
+        self.parameters = None  # Calibration object with all relevant parameters
+
+
+# TODO: replace fake testing functions with actual acquisition
+import time
+executor = model.CancellableThreadPoolExecutor(max_workers=1)
+def acquire(roas, path):
+    """
+    :param roas: (list of FastEMROA) list of acquisition regions to be acquired
+    :param path: (str) path and filename of the acquisition on the server
```

Comment:
filename of which acquisition? They are not going to have all the same name, right? 
Shouldn't this be an attribute on the RoA?

Reply:
Sorry, I forgot to update this. The function should take a single ROA, not a list. 

Indeed, we could consider making the path an attribute. The folder structure is <date>/<project_name>/<ROA name>. The ROA name is already an attribute. We need some extra information for the date and project, it could be stored in a second VA `.path` or `.filename`. Do you think that's better than passing it as an additional argument?